Pod/Native/NativeFeatureParser.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  NativeFeatureParser.swift
+//  Pods
+//
+//  Created by Marcin Raburski on 05/09/2016.
+//
+//
+
+import Foundation
+
+class NativeFeatureParser {
+    let path: NSURL
+    init(path: NSURL) {
+        self.path = path
+    }
+    
+    func parsedFeatures() -> [NativeFeature]? {
+        let manager = NSFileManager.defaultManager()
+        var isDirectory: ObjCBool = ObjCBool(false)
+        guard manager.fileExistsAtPath(self.path.path!, isDirectory: &isDirectory) else {
+            assertionFailure("The path doesn not exist '\(path)'")
+            return nil
+        }
+        
+        if isDirectory {
+            // Get the files from that folder
+            if let files = manager.enumeratorAtURL(path, includingPropertiesForKeys: nil, options: [], errorHandler: nil) {
+                return self.parseFeatureFiles(files)
+            } else {
+                assertionFailure("Could not open the path '\(path)'")
+            }
+            
+        } else {
+            if let feature = self.parseFeatureFile(path) {
+                return [feature]
+            }
+        }
+        return nil
+    }
+    
+    func parseFeatureFiles(files: NSDirectoryEnumerator) -> [NativeFeature] {
```

Comment:
parseFeatureFiles and parseFeatureFile can be marked fileprivate so it's right away clear these are only used within NativeFeatureParser